### ISSUE TITLE
PHP Modules

### ISSUE THREAD

**Issue by tpnsolutions** [virtualmin/virtualmin-gpl#347]
I've been noticing when installing Virtualmin on Ubuntu 20.04, that a number of essential PHP modules are not being installed by default. I don't remember this being an issue with CentOS, in fact it usually installed about half a dozen modules that were considered essential building blocks like: php-gd, php-curl, etc...

Frankly, up to this point I really just dismissed the missing modules as I'd just add them as a script needed them, however I felt it was time to bring this up as I think it would be ideal to have them installed during Virtualmin install phase.

Below is a list of modules I think would be useful for most, it is not exhaustive but really just a starting point...

php-bcmath
php-gd
php-imagick
php-intl
php-json
php-mbstring
php-pspell
php-ssh2
php-xml
php-xmlrpc
php-zip

Some of the packages above may actually getting installed, and I based it loosely on what was installed on an older CentOS 7 installation.

Again, something to look into, to enhance the overall installation process.

**Comment by jcameron** [comment]
Yeah, currently we install modules only as needed by PHP scripts. I suppose we could install some as part of the Virtualmin install process, but this wouldn't actually guarantee they are available if the user later installs a different PHP version.

**Comment by Ramoonus** [comment]
personally I only use php-common
and install the other ones if a script specifically asks for it

**Comment by iliajie** [comment]
The upcoming Webmin version will include an option to manage PHP extensions in the "PHP Configuration" module.
